lib/ansible/modules/system/aix_lvol.py

```diff
@@ -205,7 +216,7 @@ def main():
             lv_type=dict(type='str', default='jfs2'),
             size=dict(type='str'),
             opts=dict(type='str', default=''),
-            copies=dict(type='str', default='1'),
+            copies=dict(type='int', default=1),
```

Comment:
No, because the value is used as a string in the code, and the example was using an integer. In fact it is an integer (and not just a string) so I'd rather have the module fail with a proper error that an integer is expected, than the command fail with a dubious error because something non-integer like was added.

However, I do understand if you would prefer to keep it out of this PR.

Reply:
Ah, excellent. I do think it would be better in a separate PR, since it's a different kind of fix. Pulling it out will make this change easier to find.

Reply:
Removed.